.github/workflows/CI.yml

```diff
@@ -84,7 +85,7 @@ jobs:
       - name: Run cargo-tarpaulin
         uses: actions-rs/tarpaulin@v0.1
         with:
-          args: "--verbose --all-features --workspace --timeout 1200"
+          args: "--verbose --all-features --run-types Doctests,Tests --timeout 1200 --ignored"
```

Comment:
cargo-tarpaulin doesn't include doctests or ignored tests by default, so adding explicit `--run-types` and `--ignored`